src/OneExplorer/ConfigObject.ts

```diff
@@ -140,6 +148,10 @@ export class ConfigObj {
     this.uri = uri;
     this.rawObj = rawObj;
     this.configType = CfgType.one;
+    const ext = path.extname(uri.fsPath);
+    if (ext === EdgeTpuConfigSetting.ext) {
+      this.configType = CfgType.edgeTpu;
+    }
```

Comment:
How about tidying these by setting configType with similar conditional statement?

Reply:
```
    let configSetting: ConfigSetting;
    this.configType = CfgType.one;
    const ext = path.extname(uri.fsPath);
    if (ext === EdgeTpuConfigSetting.ext) {
      this.configType = CfgType.edgeTpu;
    }
    switch (this.configType) {
      case CfgType.edgeTpu:
        configSetting = new EdgeTpuConfigSetting();
        break;
      case CfgType.one:
      default:
        configSetting = new OneConfigSetting();
    }
    this.uri = uri;
    this.rawObj = rawObj;
```
There seems to be an issue where you can set CfgType to edgetpu after creating OneConfigSetting. What do you think about this approach?